core/src/sumeragi/network_topology.rs

```diff
@@ -190,10 +191,10 @@ impl Builder {
 }
 
 /// The ordering of the peers which defines their roles in the current round of consensus.
-#[derive(Clone, Debug, Encode, Decode, IntoSchema, Serialize)]
+#[derive(Clone, Debug, Encode, Decode, Serialize)]
```

Comment:
can you tell me if `Serialize` really required here?

Reply:
i will check